src/styles/Dashboard.module.css

```diff
@@ -0,0 +1,39 @@
+.left{
+    height: 100%;
+    margin: 0;
+  }
+
+.right{
+    height: 100%;
+    margin: 0;
+  }
+
+.container{
+    width: 100%;
+    min-height: 100vh;
+
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+
+}
+
+@media only screen and (max-width: 992px) {
+
+  .container{
+    grid-template-columns: 1fr;
+  }
+  .right{
+    width: 100%;
+    min-height: 50vh;
+  }
+
+  .left{
+    min-height: 75vh;
+  }

```

Comment:
Why do they have different heights?

Reply:
I am not sure to be honest, but I can change .right to 75, since it feels small right now.